config/samba/bundle.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM scratch
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=smb-csi-driver-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
```

Comment:
Should this channel be `preview` like [secrets-store](https://github.com/openshift/secrets-store-csi-driver-operator/blob/main/config/bundle.Dockerfile#L6) while it's still TechPreview?
If yes, then it also means renaming config/manifests/stable to config/manifests/preview and `s/stable/preview/` in the other files too.

Reply:
good point, moved to `preview/`